plugins/chat/assets/javascripts/discourse/initializers/chat-sidebar.js

```diff
@@ -196,7 +203,8 @@ export default {
               }
 
               get suffixCSSClass() {
-                return this.channel.tracking.mentionCount > 0
+                return this.channel.tracking.mentionCount > 0 ||
+                  this.channel.tracking.watchedThreadsUnreadCount > 0
```

Comment:
Makes public channels in sidebar have urgent badge when a watched thread has unreads.